understanding/captions-prerecorded.html

```diff
@@ -58,149 +56,122 @@
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#resources">Related Resources</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
-                     <li><a href="#test-rules">Test Rules</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.2.2:</span>Captions (Prerecorded) (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.2.2:</span>キャプション (収録済) (レベル A)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
                   <p>
-                     <a href="#dfn-captions">Captions</a> are provided for all <a href="#dfn-prerecorded">prerecorded</a>
-                     <a href="#dfn-audio">audio</a> content in <a href="#dfn-synchronized-media">synchronized media</a>, except when the media is a <a href="#dfn-media-alternative-for-text">media alternative for text</a> and is clearly labeled as such.
+                     <a href="#dfn-synchronized-media">同期したメディア</a>に含まれているすべての<a href="#dfn-prerecorded">収録済</a>の<a href="#dfn-audio">音声</a>コンテンツに対して、<a href="#dfn-captions">キャプション</a>が提供されている。ただし、その同期したメディアが<a href="#dfn-media-alternative-for-text">メディアによるテキストの代替</a>であって、メディアによる代替であることが明確にラベル付けされている場合は除く。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Videos can be played with captions.</dd>
+                  <dt>目標</dt>
+                  <dd>映像をキャプション付きで再生できるようにする。</dd>
```

Comment:
原文に make や let がないので、「ようにする」は不要かも?

```suggestion
                  <dd>映像をキャプション付きで再生できる。</dd>
```